File: TwoSum/TwoSum.js
var twoSum = function(nums, target) {
  for (let i = 0; i < nums.length - 1; i++) {
    let rem = target - nums[i]
    for (let j = i + 1; j < nums.length; j++) {
      if (nums[j] == rem) return [i, j]
    }
  }
  return false
};

File: TwoSum/TwoSum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        seen = {}
        for i in range(len(nums)):
            x = seen.get(target - nums[i])
            if x != None:
                return [i, x]
            seen[nums[i]] = i
        return None
